### PR TITLE
last-tab editing wp fix

### PR DESCRIPTION
**Pull request by ginold**
related to #949

### CODE REVIEW

File: c2corg_ui/templates/waypoint/edit.html [hunk 1]
```diff
@@ -40,19 +40,14 @@ <h1 ng-init="id = ${waypoint_id if waypoint_id else 0}" class="text-center">
...
     <app-progress-bar>
-      ## It won't work with ng-init
-      <span class="ng-hide">
-        {{progressBarCtrl.updateMaxSteps(waypoint.waypoint_type)}}
-        {{progressBarCtrl.waypointType = waypoint.waypoint_type}}
-      </span>
 
       <ul class="progress-bar-edit">
-        <li class="nav-step-1"ng-click="progressBarCtrl.step(1, 'waypoint',  'backwards')" ng-class="{'nav-step-selected': progressBarCtrl.currentStep == 1 }"><span class="nav-text"><span translate>waypoint</span></span><span class="bullet-container"><span class="bullet"></span></span></li>
+        <li class="nav-step-1"ng-click="progressBarCtrl.step(1, 'waypoint',  'backwards')"><span class="nav-text"><span translate>waypoint</span></span><span class="bullet-container"><span class="bullet"></span></span></li>
```
Comment (brunobesson) [inline c2corg_ui/templates/waypoint/edit.html]: [pedantic mode on]

    <li class="nav-step-1"ng-click="progressBarCtrl.step(1, 'waypoint',  'backwards')">

should be instead

    <li class="nav-step-1" ng-click="progressBarCtrl.step(1, 'waypoint',  'backwards')">

Comment (ginold) [inline c2corg_ui/templates/waypoint/edit.html]: > [pedantic mode on]

that's Alex'es job ! :P

Comment (brunobesson) [inline c2corg_ui/templates/waypoint/edit.html]: He is doing the same with everyone!
You know the story: you do to others what was done to you before, as some kind of revenge :smile:

Comment (brunobesson) [inline c2corg_ui/templates/waypoint/edit.html]: Did I mention there are also 2 spaces between `'waypoint'` and `'backwards'`? :innocent:

**Pull request merged**